services/gundeck/src/Gundeck/Push.hs

```diff
@@ -235,6 +235,9 @@ addToken (uid ::: cid ::: req ::: _) = do
                 Log.info $ "token" .= tokenText tok
                         ~~ msg (val "Invalid push token.")
                 return (Left invalidToken)
+            Left (Aws.TokenTooLong l) -> do
+                Log.info $ msg ("Push token is too long: token length = " ++ show l)
```

Comment:
remove debug logging

Reply:
Are you sure? The other errors caught here have it.